peribolos/knative.yaml

```diff
@@ -624,6 +624,12 @@ orgs:
         description: 'DEPRECATED: Development continues in https://github.com/knative/operator/'
         has_projects: true
         has_wiki: false
+      func:
```

Comment:
Shouldn't this go into `knative-sandbox.yaml` as the repositories are still ? Or is this just mirroring the sandbox configuration ?

Reply:
@rhuss it has been discussed, but not as a requirement for being a full WG. Do you think it would be better to keep everything as `knative-sandbox` for now? What I am doing here has been additive, not replacing what already exists, fwiw.

Reply:
no problem and no blocker for me.